code/modules/antagonists/heretic/magic/star_touch.dm

```diff
@@ -0,0 +1,230 @@
+/datum/action/cooldown/spell/touch/star_touch
+	name = "Star Touch"
+	desc = "Marks someone with a star mark or puts someone with a star mark to sleep for 4 seconds, removing the star mark. \
+		When the victim is hit it also creates a beam that deals a bit of fire damage and damages the cells. \
+		The beam lasts a minute, until the beam is obstructed or until a new target has been found. \
+		Star Touch can remove cosmic runes or teleport you to your Star Gazer when you have one."
```

Comment:
```suggestion
		You and your target are linked with a cosmic ray, burning them for up to a minute, or \
		until they can escape your sight. Star Touch can also remove Cosmic Runes, or teleport you \
		to your Star Gazer when used on yourself."
```

Should probably instruct people on _how_ to teleport to their Star Gazer. Additionally, I couldn't figure out the beam from this description until reading the code, so that might be clearer.

I don't know how much I love overloading this spell as an attacking spell which has an unrelated purpose of teleporting you to your minion (deleting runes at least makes more sense because it's like "an attack") but eeeeeh it's probably fine.